pkg/config/load.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"io/ioutil"
+
+	"github.com/jessevdk/go-flags"
+)
+
+type ConfigPath struct {
+	ConfigPath string `long:"config" description:"yaml config file path"`
+}
+
+type Validator interface {
+	Validate() error
+}
+
+func LoadAndValidateConfig(v Validator) error {
+	err := Load(v)
+	if err != nil {
+		return err
+	}
+	return v.Validate()
+}
+
+// Load loads config from ENV config or arguments config.
```

Comment:
is it loading from env ?

Reply:
Yes you can use env `CONFIG` or arguments `--config` . Both are supported to set path to the cfg file.